content/en/account_management/authen_mapping/_index.md

```diff
@@ -0,0 +1,498 @@
+          "id": "123e4567-e89b-12d3-a456-426655440000",
+          "type": "roles"
+        }
+      }
+    }
+  },
+  "included": [
+    {
+      "data": {
+        "id": "123e4567-e89b-12d3-a456-426655440000",
+        "type": "roles",
+        "attributes": {
+          "created_at": "timestamp",
+          "modified_at": "string",
+          "uuid": "123e4567-e89b-12d3-a456-426655440000",
+          "name": "string"
+        },
+        "relationships": {
+          "data": [
+            {
+              "id": "123e4567-e89b-12d3-a456-426655440000",
+              "type": "permissions"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "data": {
+        "id": 0,
+        "type": "saml_assertion_attributes",
+        "attributes": {
+          "id": 6,
+          "attribute_key": "string",
+          "attribute_value": "string"
+        }
+      }
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Update mapping
+
+Updates the AuthN Mapping `role`, `saml_assertion_attribute_id`, or both from a JSON body. Returns the updated AuthN Mapping.
+
+
+| Method | Endpoint path | Required payload |
+|--------|--------------|------------------|
+| `PATCH`  | `/v2/authn_mappings/{ID}`  | JSON       |
+
+##### ARGUMENTS
+
+* **`id`** [*required*, *default*=none]:
+  The UUID of the `authinmap` being updated. For more information about finding your UUID, see the [Role API documentation][1].
+* **`role_uuid`** [*required*, *default*=none]:
+  The Roles API can be used to create and manage Datadog roles, what global permissions they grant, and which users belong to them. When you create a Role, it is assigned a UUID. For more information about finding the `role_uuid` for the role you are updating, see the [Role API documentation][1].
+* **`attribute_key`** [*required*, *default*=none]:
+ The `attribute_key` and the `attribute_value` are key/value pairs defined in their SAML assertions from their Identity Providers. You can define these for your own use case. For example, `attribute_key` could be `member-of` and the `attribute_value` could be `Development`.
+* **`attribute_value`** [*required*, *default*=none]:
+  The `attribute_key` and the `attribute_value` are key/value pairs defined in their SAML assertions from their Identity Providers. You can define these for your own use case. For example, `attribute_key` could be `member-of` and the `attribute_value` could be `Development`.
+
+{{< tabs >}}
+{{% tab "Example" %}}
+
+```sh
+curl -X PATCH \
+         "https://app.datadoghq.com/api/v2/authn_mappings/{UUID}" \
+         -H "Content-Type: application/json" \
+         -H "DD-API-KEY: <YOUR_DATADOG_API_KEY>" \
+         -H "DD-APPLICATION-KEY: <YOUR_DATADOG_APPLICATION_KEY>" \
+         -d '{
+             "data": {
+                 "type": "authn_mappings",
+                 "role_uuid": "123e4567-e89b-12d3-a456-426655440000",
+                 "attributes": {
+                      "id": "123e4567-e89b-12d3-a456-426655445555",
```

Comment:
Gotta swap these, the top level `id` should be the id referring to the authn mapping, and the `role_uuid` should belong in the attributes